extract_all_binaries.js

```diff
@@ -25,149 +25,189 @@ var OS = process.platform; // e.g. linux
+  'linux-ppc64',
+  'linux-ppc64le',
+  'linux-s390',
+  'linux-s390x',
+  'linux-x64',
+  'win32-ia32',
+  'win32-x64',
+];
+var AGENTCORE_VERSION = '3.2.1';
+var APPMETRICS_VERSION = '3.0.2';
 
 var LOG_FILE = path.join(INSTALL_DIR, 'install.log');
-var logFileStream = fs.createWriteStream(LOG_FILE, {flags : 'a'});
+var logFileStream = fs.createWriteStream(LOG_FILE, { flags: 'a' });
 
-console.log = function(info) { //
-	logFileStream.write(util.format(info) + '\n');
-	process.stdout.write(util.format(info) + '\n');
+console.log = function(info) {
+  //
+  logFileStream.write(util.format(info) + '\n');
+  process.stdout.write(util.format(info) + '\n');
 };
 
 var showLegalWarning = function() {
-	/* Legal warning */
-	console.log(new Date().toUTCString());
-	console.log('********************************************************************************');
-	console.log('You are installing the Node Application Metrics monitoring and profiling module.');
-    console.log('Licensed under the Apache License, Version 2.0 (the "License")');
-    console.log('you may not use this file except in compliance with the License.');
-    console.log('You may obtain a copy of the License at');
-    console.log('');
-    console.log('http://www.apache.org/licenses/LICENSE-2.0');
-    console.log('');
-    console.log('Unless required by applicable law or agreed to in writing, software');
-    console.log('distributed under the License is distributed on an "AS IS" BASIS,');
-    console.log('WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.');
-    console.log('See the License for the specific language governing permissions and');
-    console.log('limitations under the License.');	
-	console.log('********************************************************************************');
+  /* Legal warning */
+  console.log(new Date().toUTCString());
+  console.log(
+    '********************************************************************************'
+  );
+  console.log(
+    'You are installing the Node Application Metrics monitoring and profiling module.'
+  );
+  console.log('Licensed under the Apache License, Version 2.0 (the "License")');
+  console.log(
+    'you may not use this file except in compliance with the License.'
+  );
+  console.log('You may obtain a copy of the License at');
+  console.log('');
+  console.log('http://www.apache.org/licenses/LICENSE-2.0');
+  console.log('');
+  console.log(
+    'Unless required by applicable law or agreed to in writing, software'
+  );
+  console.log(
+    'distributed under the License is distributed on an "AS IS" BASIS,'
+  );
+  console.log(
+    'WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.'
+  );
+  console.log(
+    'See the License for the specific language governing permissions and'
+  );
+  console.log('limitations under the License.');
+  console.log(
+    '********************************************************************************'
+  );
 };
```

Comment:
that's better then having every line prefixed with console.log. And it gets even better when you change the text and have to rewrap the lines. Butal when its a bunch of lines like `'one line' +`, but any text editor can do it when it uses backticks. And anyhow, this would normally be written like:
```
var licenseWarning = `
blah
blah
`;

function whatever() {
   console.log(licenseWarning)
}
```